ddtrace/settings/integration.py

```diff
@@ -44,8 +43,9 @@ def __init__(self, global_config, name, *args, **kwargs):
         analytics_enabled, analytics_sample_rate = self._get_analytics_settings()
         self.setdefault("analytics_enabled", analytics_enabled)
         self.setdefault("analytics_sample_rate", float(analytics_sample_rate))
-
-        service = get_env(name, "service", default=get_env(name, "service_name", default=None))
+        service = os.getenv(
+            "DD_" + name.upper() + "_" + "SERVICE", os.getenv("DD_" + name.upper() + "_" + "SERVICE_NAME", default=None)
```

Comment:
For consistency, I don't have strong feelings either way
```suggestion
            "DD_%s_SERVICE" % name.upper(),  default=os.getenv("DD_%s_SERVICE_NAME" % name.upper(), default=None)
```